builder/image-ros.sh

```diff
@@ -113,6 +113,9 @@ NPM_CONFIG_UNSAFE_PERM=true gitbook install
 gitbook build
 touch node_modules/CATKIN_IGNORE docs/CATKIN_IGNORE _book/CATKIN_IGNORE clever/www/CATKIN_IGNORE apps/CATKIN_IGNORE # ignore documentation files by catkin
 
+echo_stamp "Write down image version"
+sed -i "s|<!-- \$VERSION -->|$(</etc/clever_version)|g" clever/www/index.html
```

Comment:
Not thrilled about changing a VCS-controlled file "on the fly". Don't we want `git status` to report no changes on a clean image?

Reply:
Fair point.